### PR TITLE
adding 1 to prediction labels to match data labels

### PR DESCRIPTION
**Pull request by jcoll3**

### CODE REVIEW

File: HW3/classifier.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ def train_classifier(features, labels, args):
 
 def make_predictions(clf, features):
     prediction_probs = clf.predict_proba(features)
-    return np.argmax(prediction_probs, axis=-1), \
+    return np.argmax(prediction_probs, axis=-1)+1, \
```
Comment (erfannoury) [inline HW3/classifier.py]: Thank you for the fix. Can you put an space before and after the `+` operator to follow the code formatting?

Comment (jcoll3) [reply HW3/classifier.py]: Sure, one sec

**Comment by erfannoury** [comment]
LGTM.

**Comment by erfannoury** [comment]
Merged. Thanks @jcoll3 :)

**Pull request merged**